### ISSUE TITLE
Wrong row/col position for syntax error: Invalid symbol

### ISSUE THREAD

**Issue by Cnly** [clj-kondo/clj-kondo#734]
**version**

v2020.01.28-SNAPSHOT (clj-kondo-2020.01.28-20200202.202532-14.jar)

**platform**

JVM

**problem**

When a file contains a line like

```clojure
(unknown/)
```

clj-kondo always reports an ` error: Invalid symbol: unknown/.` on line 1 regardless of the actual position of the problematic code.

**repro**

The easiest way to reproduce this is the following command:

```sh
echo '(ns user)\n\n(unknown/)' | lein run -m clj-kondo.main --lint -
```

which corresponds to the following code:

```clojure
(ns user)

(unknown/)
```

Output from clj-kondo:

```
<stdin>:1:9: error: Invalid symbol: unknown/.
linting took 15ms, errors: 1, warnings: 0
```

but the correct position should be line 3.

**expected behavior**

clj-kondo should report the correct position of the syntax error.

**Comment by borkdude** [comment]
Thanks. I added the clj-together label. If this project gets funded by Clojurists Together I can do it during the funding period.

**Comment by borkdude** [comment]
Seems to be already fixed.
